src/util/config/build.js

```diff
@@ -25,6 +25,21 @@ export const schema = {
 			arg: 'asset-port',
 			env: process.env.NODE_ENV !== 'test' && 'ASSET_SERVER_PORT', // don't read env in tests
 		},
+		protocol: {
```

Comment:
i'm confused as to why these are build config changes. ideally this would be in `src/util/config/`, right?